v2/cn/ann-1019003001.md.tt2

```diff
@@ -10,16 +10,23 @@ OpenResty web platform based on NGINX and LuaJIT.
 
 The (portable) source code distribution, the Win32/Win64 binary
 distributions, and the pre-built binary Linux packages for Ubuntu,
-Debian, Fedora, CentOS, RHEL, Alpine, Amazon Linux are provided on this
-[Download page](download.html).
+Debian, Fedora, CentOS, RHEL, Alpine, OpenSUSE, Amazon Linux are provided
+on this [Download page](download.html).
 
 # Version Highlights
 
-A brife overview of the most noteable changes:
+On the i9-9900K test environment, with the simple Lua Hello world interface,
+we can increase the QPS of a single worker from 221,939 rps to 269,423 rps,
+with a 20%+ improvement.
+
+And here's a brife overview of the most noteable changes:
```

Comment:
We don't need this sentence here. All these are highlights anyway. See our previous formal version announcement for the template.